doc/migrate-to-pymongo4.rst

```diff
@@ -71,6 +71,13 @@ Removed the ``waitQueueMultiple`` keyword argument to
 ``waitQueueMultiple`` to bound queuing, limit the size of the thread
 pool in your application.
 
+The socketKeepAlive parameter is removed
+..........................................
+
+Removed the ``socketKeepAlive`` keyword argument to
+:class:`~pymongo.mongo_client.MongoClient`. PyMongo now always enables TCP
+keepalive.
```

Comment:
Can we continue linking to https://docs.mongodb.com/manual/faq/diagnostics/#does-tcp-keepalive-time-affect-mongodb-deployments here?

Reply:
Done.